build.gradle

```diff
@@ -6,7 +20,71 @@ subprojects {
   targetCompatibility = 17
 
   repositories {
-      mavenCentral()
-      jcenter()
+    mavenCentral()
+    jcenter()
+  }
+
+  configurations.all {
+    resolutionStrategy {
+      force "org.slf4j:slf4j-api:$slf4jVersion"
+      force "io.fabric8:kubernetes-model-core:$fabric8Version"
```

Comment:
I believe these two lines are irrelevant to `Static Analysis`. Please remove the irrelevant ones.